bootstrap.sh

```diff
@@ -154,14 +154,14 @@ if [ -z "$MYSQL_FLAVOR" ]; then
 fi
 case "$MYSQL_FLAVOR" in
   "MySQL56")
```

Comment:
Why not add a case for MySQL57 here?

```sh
"MySQL56" | "MySQL57")
```

In the current form it's misleading that test.go knows a "mysql57" flavor, but bootstrap.sh doesn't.

The entry in Dockerfile.mysql57 is also confusing because of that:
```
ENV MYSQL_FLAVOR MySQL56
```

Having a MySQL57 flavor would be cleaner. That we map it internally to 5.6 shouldn't bother the enduser.